src/creatures/players/player.cpp

```diff
@@ -2669,6 +2669,11 @@ BlockType_t Player::blockHit(std::shared_ptr<Creature> attacker, CombatType_t co
 }
 
 void Player::death(std::shared_ptr<Creature> lastHitCreature) {
+	if (!g_configManager().getBoolean(TOGGLE_MOUNT_IN_PZ, __FUNCTION__) && isMounted()) {
+		dismount();
```

Comment:
e se utilizássemos dessa forma? porque quando o player saísse de pz, a mount "voltaria".
```suggestion
		outfit.lookMount = 0;
```

Reply:
Teria que testar, mas notei que sem o dismount a velocidade que aumentou com a montaria não baixava ao morrer e ir pro templo. Chegar em casa eu testo ou se alguém quiser testar.

Reply:
> Teria que testar, mas notei que sem o dismount a velocidade que aumentou com a montaria não baixava ao morrer e ir pro templo. Chegar em casa eu testo ou se alguém quiser testar.

fiz essa alteração local aqui, estou buildando para testar :)

Reply:
@Nyedson na real, a função dismount() faz `defaultOutfit.lookMount = 0;` então é melhor do jeito que tá. kk